README.md

```diff
@@ -119,6 +121,24 @@ max_incremental_backups_per_full = 6
 max_backups = 7
 ```
 
+#### Scaleway S3-compatible Object Storage, full backups
```

Comment:
I'd like to keep the readme as storage provider agnostic as possible. I think it's fine to just document the `max_multipart_parts` parameter.

Reply:
Absolutely fine, thanks for the feedback